docs/deploy/cloud.md

```diff
@@ -0,0 +1,168 @@
+identity in the same account that the Lambda is deployed to. Create a new role
+that has permission to invoke your Lambdas and give it the following trust policy:
+
+```json
+{
+  "Sid": "AllowRestateCloudToAssumeRole",
+  "Effect": "Allow",
+  "Principal": {
+    "AWS": "AROAZQ3DNV5IY6KP4ODO4"
+  },
+  "Action": ["sts:AssumeRole", "sts:TagSession"],
+  "Condition": {
+    "StringEquals": {
+      "sts:ExternalId": "$ENVIRONMENT_ID"
+    }
+  }
+}
+```
+
+<Admonition type="info" title="Trust policy">
+This trust policy allows the Restate cloud `us.restate.cloud` region role ID (`AROAZQ3DNV5IY6KP4ODO4`) to assume your role, as long as it is using an ExternalId that matches your environment ID. The environment ID can be found in the UI and in the output of `restate whoami`.
+</Admonition>
+
+You can now register your Lambda through the new role:
+
+```shell
+restate dp add <LAMBDA_FUNCTION_ARN> --assume-role-arn <ROLE_ARN>
+```
+
+If something isn't working, the environment logs in the cloud UI may help
+you find the issue.
+
+## Invoking your services
+
+In the UI you can find the URL of your Restate environment. All requests must be
+authenticated using a Bearer token. We suggest that humans use their SSO token
+which is obtained by `restate cloud login`, and will be used automatically by
+the CLI. If you want to test invocations using `curl` as described in the [invoke docs](/invoke/http), you can use a local authenticated proxy to expose
+the Restate server as if it was running on your machine:
+
+```bash
+restate cloud env proxy
+curl localhost:8080/MyService/myHandler
+```
+
+### API tokens
+
+For programmatic invocation, awakeable completion, or admin API usage from
+scripts, CI pipelines, API gateways, or services that exist outside Restate,
+you can create an API key in the UI, selecting an appropriate role for your use
+case.
+
+```bash
+curl -H "Authorization: Bearer $MY_API_KEY" https://201hy10cd3h6426jy80tb32n6en.env.us.restate.cloud:8080/MyService/MyHandler
+curl -H "Authorization: Bearer $MY_API_KEY" https://201hy10cd3h6426jy80tb32n6en.env.us.restate.cloud:9070/deployments
+```
+
+To use the CLI programmatically with this token:
+
+```bash
+export RESTATE_HOST_SCHEME=https RESTATE_HOST=201hy10cd3h6426jy80tb32n6en.env.us.restate.cloud RESTATE_AUTH_TOKEN=$MY_API_KEY
+restate whoami
+```
+
+## Securing your services
+
+### HTTP
+
+Restate invokes your services over the public internet. For production use cases
+HTTPS must be used between Restate and your services. You can apply a
+certificate to the service directly, but its likely easier to use a fronting
+load balancer like an AWS NLB.
+
+It is also a good idea to secure access to your service so that only Restate can
+call it. The easiest way to do this is with our native request identity feature.
+All requests to your service will contain a signature from a public key which can be seen in the UI. It's safe to include it directly in
+your service code:
```

Comment:
Maybe a minor improvement?

```suggestion
You must secure access to your service so that only Restate can call it.
The easiest way to do this is with our native request identity feature.
All requests to your service will be signed with a unique environment-specific private
key. You can find the corresponding public key in the environment settings UI, under HTTP Services.
It is safe to include this public key directly in your service code:
```